src/bundle.c

```diff
@@ -443,6 +443,9 @@ int install_bundles(struct list *bundles, int current_version, struct manifest *
 		ret = do_staging(file, mom);
 		if (ret == 0) {
 			rename_staged_file_to_final(file);
+		} else {
+			printf("Failed to stage file(s) (ret = %d). Cannot proceed with bundle-add \n", ret);
```

Comment:
Since only a single file is being processed at this point, can we print out the file->filename that failed to stage? This is similar to the do_staging() error handling in the update path.